org.engplayer.EngPlayer.yml

```diff
@@ -0,0 +1,66 @@
+app-id: org.engplayer.EngPlayer
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: engplayer
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --filesystem=/mnt
+  - --filesystem=/run/media
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - python3-requirements.json
+
+  - name: engplayer
+    buildsystem: simple
+    build-commands:
+      - |
+        for dir in resources/locale/*/; do
+          lang=$(basename "$dir")
+          if [ -f "resources/locale/$lang/LC_MESSAGES/engplayer.po" ]; then
+            echo "Compiling language: $lang"
+            msgfmt "resources/locale/$lang/LC_MESSAGES/engplayer.po" -o "resources/locale/$lang/LC_MESSAGES/engplayer.mo"
+          fi
+        done
+      - mkdir -p /app/share/engplayer
+      - cp -r . /app/share/engplayer
+      - rm -rf /app/share/engplayer/__pycache__
+      - rm -rf /app/share/engplayer/venv
+      - rm -rf /app/share/engplayer/.git
+      - mkdir -p /app/bin
+      - echo '#!/bin/sh' > /app/bin/engplayer
```

Comment:
Never use shell commands to generate a script. Use `type: script` or directly a file.

Reply:
maybe they should also be upstream.